docs/syzbot.md

```diff
@@ -245,7 +245,20 @@ then the program needs to be built with `-m32` flag.
 If the reproducer exits quickly, try to run it several times, or in a loop.
 There can be some races involved.
 
-Exact compilers used by `syzbot` can be found here:
+Latest compiler used by syzbot is contained in `gcr.io/syzkaller/syzbot:gcc-10.2.1` docker image.
```

Comment:
[Here](https://github.com/google/syzkaller/blob/master/tools/docker/README.md) it was mentioned that some users don't have access to `gcr.io`. In that case, wouldn't it make sense to also upload the syzbot image to github and mention it here? Or is that `README.md` no longer accurate?

Reply:
So far I failed to push a new image to github registry.
I am getting `denied: permission_denied: create_package` errors when I try to push, and I don't see such permission for access tokens at all (there are only read/write/delete permissions for package).
Also what was previously called docker.pkg.github.com now called ghcr.io. Maybe there are other changes related to that switch...

Reply:
It seems that this should be done at the level of https://github.com/google

https://docs.github.com/en/packages/learn-github-packages/configuring-a-packages-access-control-and-visibility#configuring-access-to-container-images-for-an-organization